spec/multi_scan_manager/retrying_scan_helper_spec.rb

```diff
@@ -709,7 +734,46 @@ module TestCenter::Helper::MultiScanManager
         )
       end
 
-      it 'does not duplicate the resultBundlePath xcarg for pre Xcode 11 installs' do
+      it 'sends junit test_run info to the call back after an unrecoverable infrastructure failure when using Xcode 11 or later' do
```

Comment:
this is broken, not sure why yet 🤔 

<details>

```
expected {:batch => 1, :failed => [], :passing => [], :report_filepath => nil, :test_operation_failure => nil, :try_count => 1} to include {:test_operation_failure => "Launch session expired before checking in"}
Diff:
@@ -2,6 +2,6 @@
 :failed => [],
 :passing => [],
 :report_filepath => nil,
-:test_operation_failure => "Launch session expired before checking in",
+:test_operation_failure => nil,
 :try_count => 1,
```

</details

Reply:
I'm using Xcode 11.4 and it works fine for me (famous last words). How are you executing the tests? I type `rake` in the plugin's directory.

Reply:
I'm running `bundle exec rake` too

In `before(:each) do`, we set `allow(FastlaneCore::Helper).to receive(:xcode_at_least?).and_return(false)`. That would mean on one path is tested right? I'm removing that here and setting it per test, which causing the failure.

If I do the same in `master` in `it 'sends junit test_run info to the call back after an unrecoverable infrastructure failure' do`
- with `allow(FastlaneCore::Helper).to receive(:xcode_version).and_return('10')`
  - it passes
- with `allow(FastlaneCore::Helper).to receive(:xcode_version).and_return('11')`
  - it fails

Which would mean this test only works for Xcode 10 and earlier

I could be wrong and missing something, this is my first tooling work 😬